package.json

```diff
@@ -13,76 +13,67 @@
-    "src/",
-    "dist/",
-    "test/unittests.js",
-    "test/general",
-    "test/crypto"
+    "dist/"
   ],
   "scripts": {
-    "build": "grunt build --compat copy:openpgp_compat && grunt build --lightweight copy:openpgp_lightweight clean:js && grunt build",
-    "pretest": "grunt",
-    "test": "grunt test",
-    "lint": "eslint src"
+    "build": "rollup --config",
+    "build-test": "npm run build --build-only=test",
+    "prepare": "npm run build",
+    "test": "mocha --timeout 120000 test/unittests.js",
+    "start": "http-server",
+    "prebrowsertest": "npm run build-test",
+    "browsertest": "npm start -- -o test/unittests.html",
+    "coverage": "nyc npm test",
+    "lint": "eslint 'src/**/*.js' 'test/crypto/**/*.js'",
+    "docs": "jsdoc --configure .jsdocrc.js --destination docs --recurse README.md src",
+    "preversion": "rm -rf dist docs node_modules && npm install && npm test",
+    "version": "npm run docs && git add -A docs",
+    "postversion": "git push && git push --tags && npm publish"
   },
   "devDependencies": {
-    "babel-core": "^6.26.3",
-    "babel-plugin-syntax-async-functions": "^6.13.0",
-    "babel-plugin-transform-async-to-generator": "^6.24.1",
-    "babel-plugin-transform-regenerator": "^6.26.0",
-    "babel-plugin-transform-remove-strict-mode": "0.0.2",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-es2015-mod": "^6.6.0",
-    "babel-preset-es3": "^1.0.1",
-    "babelify": "^8.0.0",
-    "browserify-derequire": "^0.9.4",
+    "@mattiasbuelens/web-streams-adapter": "0.1.0-alpha.5",
+    "@rollup/plugin-commonjs": "^11.1.0",
+    "@rollup/plugin-node-resolve": "^7.1.3",
+    "@rollup/plugin-replace": "^2.3.2",
+    "asmcrypto.js": "github:openpgpjs/asmcrypto#475cffa5ccb2cf2556427056679414acf3610d1b",
+    "babel-eslint": "^10.1.0",
+    "bn.js": "^4.11.8",
+    "buffer": "^5.0.8",
     "chai": "^4.1.2",
     "chai-as-promised": "^7.1.1",
-    "core-js": "^2.5.3",
+    "elliptic": "github:openpgpjs/elliptic#ab7d8268c60b6abeb175841c578c224ac5b2d279",
+    "email-addresses": "3.1.0",
     "eslint": "^4.17.0",
     "eslint-config-airbnb": "^16.1.0",
     "eslint-config-airbnb-base": "^12.1.0",
-    "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-chai-friendly": "^0.5.0",
-    "grunt": "^1.1.0",
-    "grunt-browserify": "^5.3.0",
-    "grunt-contrib-clean": "~1.1.0",
-    "grunt-contrib-connect": "~1.0.2",
-    "grunt-contrib-copy": "~1.0.0",
-    "grunt-contrib-watch": "^1.1.0",
-    "grunt-header": "^1.1.0",
-    "grunt-jsbeautifier": "^0.2.13",
-    "grunt-jsdoc": "^2.2.1",
-    "grunt-mocha-test": "^0.13.3",
-    "grunt-simple-nyc": "^3.0.1",
-    "grunt-terser": "^0.1.0",
-    "grunt-text-replace": "~0.4.0",
-    "gruntify-eslint": "^4.0.0",
+    "eslint-plugin-import": "^2.8.0",
+    "hash.js": "^1.1.3",
+    "http-server": "^0.12.3",
+    "jsdoc": "github:twiss/jsdoc#0f1816eb4553856647b4ca9561b9307b11ec4f9e",
```

Comment:
is using fork really required?

Reply:
The original copies the entire source code into the docs directory. I wanted to make it smaller. See https://github.com/jsdoc/jsdoc/pull/1770. I'll move the fork from `twiss` to `openpgp` before merging.